airflow-core/src/airflow/ui/src/components/DisplayMarkdownButton.tsx

```diff
@@ -48,14 +51,24 @@ const DisplayMarkdownButton = ({
         open={isDocsOpen}
         size="md"
       >
-        <Dialog.Content backdrop>
-          <Dialog.Header bg="info.muted">
-            <Heading size="xl">{header}</Heading>
-            <Dialog.CloseTrigger closeButtonProps={{ size: "xl" }} />
-          </Dialog.Header>
-          <Dialog.Body alignItems="flex-start" as={VStack} gap="0">
-            <ReactMarkdown>{mdContent}</ReactMarkdown>
-          </Dialog.Body>
+        <Dialog.Content
+          backdrop
+          style={{
+            maxHeight: "none",
+            maxWidth: "none",
+            padding: 0,
+            width: "auto",
+          }}
```

Comment:
We don't need to use `style` here. We can inline this, and use Chakra interface.